app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class UserController extends Controller
+{
+
+    function __construct()
+    {
+        $this->middleware('check_admin');
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+        return response()->json(['test' => 'test']);
+    }
```

Comment:
```suggestion
    {
        //
    }
```
Sao lại đẩy code test lên thế này???